cli/popper/commands/cmd_ci.py

```diff
@@ -15,6 +15,22 @@
 - git clone --recursive https://github.com/systemslab/popper /tmp/popper
 - export PATH=$PATH:/tmp/popper/cli/bin
 script: popper run
+"""
+    },
+    'circle': {
+        './.circleci': """
```

Comment:
the file is `./.circleci/config.yml` as per the [official docs](https://circleci.com/docs/2.0/#adding-a-yml-file)